### PR TITLE
[bitnami/nginx-intel] Remove nginx-ldap-auth-daemon dependency for Bitnami Nginx Intel Chart

### PR DESCRIPTION
**Pull request by mdhont**
Signed-off-by: michield <michield@vmware.com>

Description of the change

Remove nginx-ldap-auth-daemon dependency from Bitnami Nginx-intel Chart.

### CODE REVIEW

**Review by carrodher — COMMENTED** [review]
I would add a note in the Upgrade section describing what is the reason behind a major version bump. Since this is a new chart, probably you will need to create the whole section. You can check https://github.com/bitnami/charts/tree/master/bitnami/nginx#upgrading as example

**Review by carrodher — APPROVED** [review]
LGTM

File: bitnami/nginx-intel/templates/deployment.yaml [hunk 1]
```diff
@@ -193,76 +193,21 @@ spec:
           resources: {{- toYaml .Values.resources | nindent 12 }}
           {{- end }}
           volumeMounts:
-            {{- if or .Values.serverBlock .Values.existingServerBlockConfigmap .Values.ldapDaemon.enabled }}
+            {{- if or .Values.serverBlock .Values.existingServerBlockConfigmap }}
             - name: nginx-server-block-paths
               mountPath: /opt/bitnami/nginx/conf/server_blocks
```
Comment (joancafom) [inline bitnami/nginx-intel/templates/deployment.yaml:198]: The `nginx-server-block-paths` was introduced as a consequence of LDAP addition. This means that it should also be removed and update the logic accordingly.

Refer to https://github.com/bitnami/charts/pull/2608/files#r427295257 for further details.

File: bitnami/nginx-intel/templates/server-block-configmap.yaml [hunk 2]
```diff
@@ -5,7 +5,6 @@ metadata:
   labels: {{- include "common.labels.standard" . | nindent 4 }}
 data:
   server-blocks-paths.conf: |-
-    include  "/opt/bitnami/nginx/conf/server_blocks/ldap/*.conf";
     include  "/opt/bitnami/nginx/conf/server_blocks/common/*.conf";
```
Comment (joancafom) [inline bitnami/nginx-intel/templates/server-block-configmap.yaml:8]: This should be removed

**Review by joancafom — CHANGES_REQUESTED** [review]
Great PR! I added a couple of comments, could you please review them?

**Review by joancafom — DISMISSED** [review]
LGTM!

**Pull request merged**